theano/sparse/tests/test_basic.py

```diff
@@ -2965,7 +2965,8 @@ def structured_function(*args):
     sparse.trunc,
     np.trunc,
     test_dtypes=[m for m in sparse.all_dtypes
-                 if not m in sparse.complex_dtypes])
+                 if not m in sparse.complex_dtypes],
+    grad_test=False)
```

Comment:
gh-5693 don't fix this particular test. Personaly, I think that messing with the input value for the tests is an easy and good fix.

@abergeron What else do you think we could do?

I put in https://github.com/Theano/Theano/pull/5702 the commit that we agree on for rc4